core/specfem/assembly/CMakeLists.txt

```
@@ -20,6 +20,13 @@ target_link_libraries(
         ${BOOST_LIBS}
 )
 
+if (__INTEL_LLVM__)
+    set_target_properties(assembly PROPERTIES
+            # UNITY_BUILD OFF
+            UNITY_BUILD_BATCH_SIZE 2
+    )
+endif()
```

```suggestion
set_target_properties(assembly PROPERTIES
    # UNITY_BUILD $<NOT:$<BOOL:${__INTEL_LLVM__}>>
    UNITY_BUILD_BATCH_SIZE $<IF:$<BOOL:${__INTEL_LLVM__}>,2,8>
)
```

Syntax is `$<IF:condition,true-value,false-value>` and `$<BOOL:string>` is explained here: https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#genex:BOOL